v0/util/index.js

```diff
@@ -344,6 +348,19 @@ const getValueFromMessage = (message, sourceKey) => {
   return null;
 };
 
+// get a field value from message.
+// if sourceFromGenericMap is true get its value from GenericFieldMapping.json and use it as sourceKey
+// else use sourceKey from `data/message.json` for actual field precedence
+// Example usage: getFieldValueFromMessage(message, "userId",true)
+//                This will return the first nonnull value from
+//                ["userId", "traits.userId", "traits.id", "context.traits.userId", "context.traits.id", "anonymousId"]
+const getFieldValueFromMessage = (message, sourceKey) => {
+  sourceKey = MESSAGE_MAPPING[sourceKey];
+  if (sourceKey) {
+    return getValueFromMessage(message, sourceKey);
+  }
```

Comment:
```suggestion
  const sourceKeyMap = MESSAGE_MAPPING[sourceKey];
  if (sourceKeyMap) {
    return getValueFromMessage(message, sourceKeyMap);
  }
  return null;
```

Can we do this instead of adding an `eslint-disable`? 
More reading: https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/

Reply:
Yeah we can do that